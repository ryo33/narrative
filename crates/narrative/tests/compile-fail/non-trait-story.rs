#[narrative::story("This should fail because it's not a trait")]
struct NotATrait {
    field: String,
}

fn main() {}
