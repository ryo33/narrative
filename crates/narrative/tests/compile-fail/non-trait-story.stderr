error: expected `trait`
 --> tests/compile-fail/non-trait-story.rs:2:1
  |
2 | struct NotATrait {
  | ^^^^^^
